docs/testing.md

```diff
@@ -39,20 +39,54 @@ They generally follow TPCH but are not guaranteed to be the same.
 
 ## Unit tests
 
-Unit tests exist in the tests directory. This is unconventional and is done so we can run the tests
-on the final shaded version of the plugin. It also helps with how we collect code coverage.
+Unit tests exist in the [here](tests) directory. This is unconventional and is done so we can run
+ the tests on the final shaded version of the plugin. It also helps with how we collect code coverage.
+
+in order to run the unit-tests follow these steps
+1. issue the maven command to run the tests with `mvn test`. this will run all the tests
+2. to run individual tests append `-dwildcardsuites=<comma separated list of wildcard suite names to execute>` to the above command 
```

Comment:
These are not steps.  They are options and I don't think we need a list to express them.